src/css/home-page.css

```
@@ -166,4 +166,24 @@ article{
 .logo{
     width: 70px;
     vertical-align: text-top;
+}
+
+
+#popInInfoDiv {
```

```suggestion
#pop-in-info-div {
```
or, we needs to change all attributes from `xx-xx` to `xxXx`

---

Adopted the kebab-case